src/internals/devices/ensureDeviceApp.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import invariant from 'invariant'
+import CommNodeHid from '@ledgerhq/hw-transport-node-hid'
+import type Transport from '@ledgerhq/hw-transport'
+import type { IPCSend } from 'types/electron'
+import getAdressFromCurrency from './getAddressForCurrency'
+
+export default async (
+  send: IPCSend,
+  {
+    currencyId,
+    devicePath,
+    accountPath,
+    accountAddress,
+    ...options
+  }: {
+    currencyId: string,
+    devicePath: string,
+    accountPath: ?string,
+    accountAddress: ?string,
+  },
+) => {
+  try {
+    invariant(currencyId, 'currencyId is defined')
```

Comment:
`is not defined` ?

Reply:
It's unclear to me if the comment here should be "what should happen" (an "invariant") or the message when it fails

Reply:
yeah i think you are correct, i'll fix it.